### PR TITLE
[tesings] More workflow tests

### PR DESCRIPTION
**Pull request by cyanglaz**
Add tests:
1. Check if the back button from the first check up page navigates the app to the home page.
2. Check if the delete user information button navigate the app to the first page.

### CODE REVIEW

File: test_driver/smoke_workflow_test.dart [hunk 1]
```diff
@@ -59,4 +72,13 @@ void main() {
   tearDown(() async {
     await driver?.close();
   });
+
+  test('User can tap on delete data and back to the first page', () async {
```
Comment (jonahwilliams) [inline test_driver/smoke_workflow_test.dart:72]: keep the tearDown at the end

Comment (cyanglaz) [reply test_driver/smoke_workflow_test.dart:72]: done! I also updated the tool tip message for the close button because it is not debug only.

**Review by jonahwilliams — APPROVED** [review]
LGTM

**Comment by dnfield** [comment]
Same here, why no CI checks on this PR?

**Pull request merged**